Use stat, fstat, and lstat structures and let the C library handle the
related structures and system calls.

Upstream: N/A, unresponsive contact email address.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
--- a/ncftp/syshdrs.h
+++ b/ncftp/syshdrs.h
@@ -266,18 +266,6 @@
 #	define Stat WinStat64
 #	define Lstat WinStat64
 #	define Fstat WinFStat64
-#elif ((defined(HAVE_LONG_LONG)) && (defined(_LARGEFILE64_SOURCE)) && (defined(HAVE_STAT64)) && (defined(HAVE_STRUCT_STAT64)))
-#	define Stat stat64
-#	ifdef HAVE_FSTAT64
-#		define Fstat fstat64
-#	else
-#		define Fstat fstat
-#	endif
-#	ifdef HAVE_LSTAT64
-#		define Lstat lstat64
-#	else
-#		define Lstat lstat
-#	endif
 #else
 #	define Stat stat
 #	define Fstat fstat
